variables.tf

```diff
@@ -35,6 +35,12 @@ variable "subnets" {
   default     = []
 }
 
+variable "elasticache_subnet_group" {
+  type        = "string"
+  description = "Subnet group ID for the Elastic cache instance"
```

Comment:
It feels like we're conflating terms here. Is this an "ID" or a "Name"? Since the upstream setting is `name`, let's stick to that.